ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/item/complex/SurveyDetail.java

```diff
@@ -93,6 +102,7 @@ public void writeToParcel(Parcel dest, int flags) {
         dest.writeDouble(this.angle);
         dest.writeDouble(this.overlap);
         dest.writeDouble(this.sidelap);
+        dest.writeByte((byte)(this.lockOrientation?1:0));
```

Comment:
@TShapinsky parcelling of new field(s) should be done after any pre-existing field(s). This is done to avoid incompatibilities with an older version of the library (i.e: older version might expect the parcelable `cameraDetail` field to start where you now have the `lockOrientation`). When you add it to the end, the older version will only parse as far as it's aware, and ignore all the new additions, maintaining compatibility.